base/applications/rapps/loaddlg.cpp

```diff
@@ -764,6 +808,15 @@ DWORD WINAPI CDownloadManager::ThreadFunc(LPVOID param)
         if (bCancelled)
             goto end;
 
+        if (!dwContentLen)
+        {
+            // set progress bar to 100%
+            SetProgressMarquee(Item, FALSE);
```

Comment:
How this sets progressbar to 100%?

Reply:
It sets total size to current downloaded size on next line: `dwContentLen = dwCurrentBytesRead;`

Reply:
OK